vagrant.go

```diff
@@ -110,7 +110,7 @@ func (v *Vagrant) setup(start bool, env string, numNodes int) error {
 		}()
 	}
 
-	v.expectedNodes = numNodes
+	v.expectedNodes = numContivNodes+numIgnoreNodes 
```

Comment:
Hmm ok so here is the problem . The code in vagrantssh assumes num nodes to be contiv_nodes only. Hence the ignore nodes.